src/PatientChecking/PatientCheckIn.DataAccess/Models/Address.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+
+#nullable disable
+
+namespace PatientCheckIn.DataAccess.Models
+{
+    public partial class Address
+    {
+        public int AddressId { get; set; }
+        public int TypeAddress { get; set; }
+        public string Address1 { get; set; }
```

Comment:
As our last discussion, this should be renamed to `StreetLine`

Reply:
I renamed Address1 to StreetLine